src/test/e2e/api/admin/favorites.e2e.test.ts

```diff
@@ -122,28 +115,6 @@ beforeEach(async () => {
     await loginRegularUser();
 });
 
-test('should be favorited in project endpoint', async () => {
-    const featureName = 'test-feature';
-    await createFeature(featureName);
-    await favoriteFeature(featureName);
-    await favoriteProject();
-
-    const { body } = await app.request
-        .get('/api/admin/projects/default')
```

Comment:
Should we still keep these tests and replace the url with `get('/api/admin/projects/default/overview')` ?